docs/README.md

```diff
@@ -382,6 +382,213 @@ NGINX Agent isn’t limited to NGINX Plus; it can also be installed into NGINX O
 
     Note that this instance has a different set of configuration recommendations than the vanilla NGINX Plus instance did. Package maintainers may ship NGINX with their own sets of defaults, which may or may not align with best practices. NGINX One provides a centralized view of such recommendations across the organization.
 
+## Lab 5: Config Sync Group
+
+We can group multiple NGINX instances into a Config Sync Group where all instances within this group use an identical configuration. This lab will go through using this feature.
+
+### Creating Config Sync Group
+
+1. Go to the NGINX One console.
+
+1. From the left menu in the "Manage" section, click on "Config Sync Groups".
+
+1. Click on "Add Config Sync Group".
+
+    ![Add Config Sync Group](media/lab5-1.png)
+
+1. On the right, provide a name then select "Create" on the bottom. Your new item is now created.
+
+    ![New Config Sync Group](media/lab5-2.png)
+
+You can now explore your Config Sync Group by selecting your item. After you select it, there are two tabs named "Details" and "Configuration". The configuration defined in this group will be used as the one NGINX config to use for all NGINX instances in this group. Notice this is empty when you first create a Config Sync Group. There are two ways to handle the initial configuration.
+
+**Option 1:** When you add the first NGINX instance, the config from the NGINX instance will be used as the config for your Config Sync Group. We will be using this option for this lab.
+
+**Option 2.** You can manually define it before you add any NGINX instances.
+
+1. Click the "Configuration" tab then click on "Edit Configuration"
+
+    ![Manually Create Initial Configuration](media/lab5-3.png)
+
+1. An editor window appears that allows you to define the configuration to use for the NGINX instances.
+
+    ![Manual Configuration](media/lab5-4.png)
+
+1. You can also add files by clicking on "Add File" but this limited to `/etc/nginx/`.
```

Comment:
Please convert all single backtick references to use bold text instead. The lab framework doesn't support inline code blocks yet.

Reply:
Okay, sure. All references fixed